templates/winthrop_staging/local_settings.py

```diff
@@ -0,0 +1,119 @@
+# This file is exec'd from settings.py, so it has access to and can
+# modify all the variables in settings.py.
+# If this file is changed in development, the development server will
+# have to be manually restarted because changes will not be noticed
+# immediately.
+
+
+# Add Google Analytics
+INCLUDE_ANALYTICS = True
+
+# Make these unique, and don't share it with anybody.
+SECRET_KEY = "{{ secret_key }}"
+
+# username for accessing GeoNames API
+GEONAMES_USERNAME = '{{ geonames_username }}'
+# mapbox access token
+MAPBOX_ACCESS_TOKEN = '{{ mapbox_token }}'
+
+# Database
+# https://docs.djangoproject.com/en/1.10/ref/settings/#databases
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.mysql',
+	'NAME': '{{ db_name }}',
+	'USER': '{{ db_username }}',
+	'PASSWORD': '{{ db_password}}',
+	'HOST': '{{ db_host }}',
+	'PORT': '3306',
+	'OPTIONS': {
+		'init_command': "SET sql_mode='STRICT_TRANS_TABLES'",
+	}
+    }
+}
+
+SOLR_CONNECTIONS = {
+    'default': {
```

Comment:
Shouldn't these be variables?

Reply:
I considered that approach. I think looking at a similar situation with Derrida, I will do just that. I wanted to feel my way around this one first.